### DISCUSSION TITLE
Last update broke something

### DISCUSSION THREAD

**Discussion by AnalyticsXXI** [lllyasviel/Fooocus#3002] [Q&A] [answered]
Last update, that which added "Hyper-SD" preset to the interface, made some checkpoints generate noise instead of images. For example, this one https://civitai.com/models/331116/pony-faetality — it stopped to work right after I noticed those new changes in interface.

**Comment by xhoxye** [comment]
Hyper-SD may not work with pony models?
Actually, CLIP Skip=1 doesn't work with the pony model

**Reply by AnalyticsXXI** [reply]
> Hyper-SD may not work with pony models?

I didn't swap to Hyper-SD. I just noticed that interface changed a bit, and it stopped to work.

**Reply by mashb1t** [reply]
@AnalyticsXXI I have not tested pony models, but there should not be any issue when using them. Please open an issue and provide all necessary information to be able to properly debug this. Thanks!

**Comment by xhoxye** [comment, marked as answer]
You need to use CLIP Skip=2
![QQ截图20240527015721](https://github.com/lllyasviel/Fooocus/assets/129571231/a81c40fe-78ae-4e9f-8705-d16b6a349ce9)
![QQ截图20240527015707](https://github.com/lllyasviel/Fooocus/assets/129571231/9a44b277-cd22-40fa-871d-1521ee44e899)
![QQ截图20240527015654](https://github.com/lllyasviel/Fooocus/assets/129571231/2ef028a7-db07-4cc9-9378-469e5ea74ba3)

**Reply by AnalyticsXXI** [reply]
Yes, right! With CLIP skip = 2, it works.

**Reply by DreadFox** [reply]
Thanks! That's fixed for me ;)

**Comment by DreadFox** [comment]
FYI I'm having to go to CLIP skip = 3 for some pony merges, though.

**Comment by Rogala** [comment]
I always try the old promt with identical settings in the new version and the result surprised me.
![2024-05-26_21-59-33_2049](https://github.com/lllyasviel/Fooocus/assets/7437582/9c828bc2-cea7-4477-8bbd-838585adcedc)

When I changed CLIP Skip = 2 everything came out identical, what is the point of this setting and why does it interfere now?
![Снимок экрана 2024-05-26 220631](https://github.com/lllyasviel/Fooocus/assets/7437582/14f596cb-b9fe-4e4f-9684-3e4551a1c508)

`{
"prompt": "A cute dragon with big eyes, shimmering rainbow scales, small horns, wings, and claws. The appearance is friendly and magical",
"negative_prompt": "",
"prompt_expansion": "A cute dragon with big eyes, shimmering rainbow scales, small horns, wings, and claws. The appearance is friendly and magical, vivid, detailed, very inspirational, beautiful, dynamic, lively, intricate, elegant, thought, cinematic, highly detail, color spread, amazing light, magic, glowing, rich deep complex, fabulous epic composition, dramatic ambient, scenic, professional",
"styles": "['Fooocus V2', 'Fooocus Enhance', 'Fooocus Sharp']",
"performance": "Lightning",
"resolution": "(960, 1024)",
"guidance_scale": 1.0,
"sharpness": 0.0,
"adm_guidance": "(1.0, 1.0, 0.0)",
"base_model": "juggernautXL_v8Rundiffusion.safetensors",
"refiner_model": "None",
"refiner_switch": 1.0,
"adaptive_cfg": 1.0,
"clip_skip": 2,
"sampler": "euler",
"scheduler": "sgm_uniform",
"vae": "Default (model)",
"seed": "3507431457051186204",
"lora_combined_1": "sd_xl_offset_example-lora_1.0.safetensors : 0.1",
"lora_combined_2": "sdxl_lightning_4step_lora.safetensors : 1.0",
"metadata_scheme": false,
"version": "Fooocus v2.4.0"
}`

**Comment by mashb1t** [comment]
See https://github.com/lllyasviel/Fooocus/discussions/3003#discussioncomment-9563985, sorry for the inconvenience. I'm working on a fix for clip_skip.
